spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/encryption/EncryptionController.java

```diff
@@ -103,7 +103,7 @@ public String encrypt(@RequestBody String data, @RequestHeader("Content-Type") M
 	@PostMapping("/encrypt/{name}/{profiles}")
 	public String encrypt(@PathVariable String name, @PathVariable String profiles, @RequestBody String data,
 			@RequestHeader("Content-Type") MediaType type) {
-		TextEncryptor encryptor = getEncryptor(name, profiles, "");
+		TextEncryptor encryptor = getEncryptor(name, profiles, data);
```

Comment:
Shouldn't we be passing `input` instead?  Thats what the original code used

Reply:
done in commit e7669e1b and use retrieved encryptor to encrypt message here https://github.com/spring-cloud/spring-cloud-config/pull/2253/files#diff-f18a83930fb437413479ed50adc18dedbf115b37e560d8481185a796c24444d8R111